manifests/2.2.0/opensearch-2.2.0.yml

```diff
@@ -35,3 +35,9 @@ components:
     checks:
       - gradle:publish
       - gradle:properties:version
+  - name: k-NN
+    repository: https://github.com/opensearch-project/k-NN.git
+    ref: '2.x'
+    checks:
+      - gradle:publish
+      - gradle:properties:version
```

Comment:
Please make sure you are copying from previous release manifest.
k-NN should be similar to this one:
```
- name: k-NN
    repository: https://github.com/opensearch-project/k-NN.git
    ref: tags/2.1.0.0
    platforms:
      - darwin
      - linux
    checks:
      - gradle:properties:version
      - gradle:dependencies:opensearch.version
```

Reply:
Thanks, updated